sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/eng/mgmt/mgmtmetadata/sql_resource-manager.txt

```diff
@@ -0,0 +1,14 @@
+﻿Installing AutoRest version: latest
```

Comment:
This metadata file should be placed at `{repo-root}/eng/mgmt/mgmtmetadata`

Reply:
Oh I manually copy and paste it to here...I'll move it back to {repo-root}/eng/mgmt/mgmtmetadata in next push.

Reply:
updated.